### PR TITLE
[ADMIN] admin logs to stuff that should have it

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Added an admin log to viewing variables, which is weird that there wasn't one

Added a `message admins` & admin log to renaming something via view variables panel, which is weird that there wasn't one

### CODE REVIEW

File: code/modules/admin/view_variables/view_variables.dm [hunk 1]
```diff
@@ -80,6 +80,8 @@
 			names += V
 	sleep(0.1 SECONDS)//For some reason, without this sleep, VVing will cause client to disconnect on certain objects.
 
+	log_admin("[key_name(usr)] viewed the variables of [D].")
```
Comment (patpol4) [inline code/modules/admin/view_variables/view_variables.dm:83]: this will spam

Comment (ynot01) [reply code/modules/admin/view_variables/view_variables.dm:83]: no more than other admin logs, it is important for admin investigations

Comment (ynot01) [reply code/modules/admin/view_variables/view_variables.dm:83]: resolved in discord

Comment (patpol4) [reply code/modules/admin/view_variables/view_variables.dm:83]: thought it would message each time, logs are good

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**